forEach.js

```diff
@@ -11,3 +11,48 @@ function cb(item, index, arr){
 };
 
 customForEach(family, cb);
+
+// Slice
+const array = ["t", "e", "s", "t"];
+
+function slice(arr,start,end) {
+  const finish = end || arr.length;
+  const newArr = [];
+  for(let i = start; i < finish; i++){
+    newArr.push(arr[i]);
+  }
+  return newArr;
+};
+ 
+console.log('Slice', slice(array, 0));
+
+// IndexOf
+function indexOf(item) {
+  let result = -1;
+  for (let i = 0; i <= array.length; i++) {
+    if (array[i] === item) {
+      result = i;
+    }
+  }
+  return result;
+}
+
+console.log(indexOf("s"));
+
+//Concat
+
+function concat(...arrays) {
+  const arr = [];
+  for(let i = 0; i < arrays.length; i++) {
+    const el = Array.isArray(arrays[i]);
+    if(el === true){
```

Comment:
el-y arden true kam false-a, imast chka stugelu
```suggestion
    if(el){
```